File: pset1:C_programming/marioMore/mario.c

#include <stdio.h>

int main(void)
    // Prompt user input for height of the pyramid 

{  int n;
    do 
    {
        n= get_int("Pyramid's Height: ");
    }
    while(n<0 || n>23);
    // Making rows - represented by the variable "i"
    for (int i= 1; i<=n; i++)
   
    {
        for (int j=1; j<=n-i; j++)
        {
            printf(" ");
        }
        for (int j=1; j<= i; j++)
        {
            printf("#");
        }
        for (int j=1; j<=2; j++)
        {
            printf(" ");
        }
         for (int j=1; j<= i; j++)
        {
             printf("#");
        }
        
        printf("\n");
    }
}

File: pset6:Python/mario/more/mario.py
from cs50 import get_int
# take user input
while True:
    height= get_int("Height: ")
    if height>0:
        break
# making hash figure using nested for loop
for row in range (1, height+1):
    for spc in range (0, height-row):
        print(" ", end="")
    for col in range (0, row):
        print("#", end="")
    print("  ", end="")
    for col2 in range (0, row):
        print("#",end="")
    print()